libs/shared/infrastructure/mongoose/src/lib/database.module.spec.ts

```
@@ -0,0 +1,64 @@
+        ConfigModule.forRoot({
+          load: [databaseConfig],
+        }),
+        DatabaseModule,
+      ],
+    }).compile();
+
+    expect(module).toBeDefined();
+  });
+
+  it('should use default values when environment variables are not set', async () => {
+    delete process.env['DATABASE_HOST'];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)